gazebo/content.md

```diff
@@ -0,0 +1,112 @@
+# What is [Gazebo](http://www.gazebosim.org/)?
+
+Robot simulation is an essential tool in every roboticist's toolbox. A well-designed simulator makes it possible to rapidly test algorithms, design robots, and perform regression testing using realistic scenarios. Gazebo offers the ability to accurately and efficiently simulate populations of robots in complex indoor and outdoor environments. At your fingertips is a robust physics engine, high-quality graphics, and convenient programmatic interfaces. Best of all, Gazebo is free with a vibrant community.
+
+> [wikipedia.org/wiki/Gazebo_simulator](https://en.wikipedia.org/wiki/Gazebo_simulator)
+
+[%%LOGO%%](http://www.gazebosim.org/)
+
+# How to use this image
+
+## Create a `Dockerfile` in your Gazebo project
+
+	FROM gazebo:gzserver5
+	# place here your application's setup specifics
```

Comment:
Where users should mount their files for gazebo is described in detail under the volumes section. I wouldn't recommend users bake-in worlds models or robot. sdf files into image itself using the `ADD` argument in the Dockerfile. Those files are usually dynamic in nature as the user tweaks or iterates designs during runtime.